crates/matrix-sdk/src/error.rs

```diff
@@ -379,6 +381,10 @@ pub enum Error {
     /// Backups are not enabled
     #[error("backups are not enabled")]
     BackupNotEnabled,
+
+    /// An error coming from the send queue subsystem.
+    #[error(transparent)]
+    SendQueueError(#[from] RoomSendQueueError),
```

Comment:
With my suggestion below, you can likely remove this…

Reply:
Can this not be removed, now?

Reply:
Sorry, I completely missed this in the previous pass, was literally working on removing it 👍